build.gradle

```diff
@@ -24,7 +24,7 @@ dependencies {
     compile "com.google.code.gson:gson:2.2.4"
     compile "com.google.guava:guava:19.0"
     compile "org.slf4j:slf4j-api:1.7.7"
-    compile "com.launchdarkly:okhttp-eventsource:0.1.0"
+    compile "com.launchdarkly:okhttp-eventsource:0.1.2"
```

Comment:
Let's publish a snapshot of this so we can get builds passing / push to heroku for testing.